ui-v2/app/components/topology-metrics/index.js

```diff
@@ -12,41 +12,65 @@ export default class TopologyMetrics extends Component {
 
   // =methods
   drawDownLines(items) {
-    return items.map(item => {
-      const dimensions = item.getBoundingClientRect();
-      const dest = {
-        x: this.centerDimensions.x,
-        y: this.centerDimensions.y + this.centerDimensions.height / 4,
-      };
-      const src = {
-        x: dimensions.x + dimensions.width,
-        y: dimensions.y + dimensions.height / 2,
-      };
+    let order = [null, 'allow', 'deny'];
```

Comment:
A random `let` makes an appearance! 😂  You usually use `const` everywhere (where you aren't changing the reference), should this be `const` also?

Reply:
😂 no clue why I thought I'd need them